### DISCUSSION TITLE
Issue in creating order

### DISCUSSION THREAD

**Discussion by manaver** [swellstores/community#479] [Help] [answered]
I am creating a travel website and i am using swell for it. There is alot of problem using swell for travel website but somehow i overcome the issues. But still it is not good.  Here's the link of the website: [https://travel-fever.vercel.app/](https://travel-fever.vercel.app/) .
My issue is , is there's any way to create order without cart??
Because using cart is not suitable for travel package website..
i.e. user can buy only 1 item at a time.
so to overcome this i am using this: 
`
    const addItem = async (item: CartItem) => {
        await swell.cart.setItems([])
        const cart = await swell.cart.addItem(item)
        //redirecting user to the cart checkout page
        if (cart?.checkout_url) {
            router.push(cart?.checkout_url);
        }
        getCart();
    }
`
It is working fine locally but i don't know why it stops working in hosting??
and this is the url of my repo of this project: [https://github.com/manaver/corizo-intern-next-travel](https://github.com/manaver/corizo-intern-next-travel)

**Comment by sags95** [comment, marked as answer]
With the frontend api it would have to go through a cart (since it follows a typical flow), however it is possible to directly create orders with the Backend API https://developers.swell.is/backend-api/orders/create-an-order

Are you seeing any errors in the hosted app? This might shed some light on what's going wrong.

We also have Purchase Links that could be used: https://www.swell.is/help/products/purchase-links

**Reply by logeshswell** [reply]
> It is working fine locally but i don't know why it stops working in hosting??

I think you need to check the debug logs from your hosting provider since it is working fine locally.
